lib/satellite/warden_strategy.rb

```diff
@@ -52,5 +52,5 @@ def user_class
 
 Warden::Manager.serialize_from_session do |serialized|
   class_name, id = serialized
-  class_name.constantize.find(id)
+  class_name.constantize.find_by(id: id)
```

Comment:
this would throw an active record not found error when we had a deleted user still in our session. Deleted users remained in the session because it wasn't cleared from a log out.

with this change,

`authenticate_user!` needs to reauthenticate because the `current_user` is anonymous. Anonymous users are not return false for registered.